lerobot/common/policies/act/modeling_act.py

```diff
@@ -521,9 +521,10 @@ def forward(self, batch: dict[str, Tensor]) -> tuple[Tensor, tuple[Tensor, Tenso
 class ACTEncoder(nn.Module):
     """Convenience module for running multiple encoder layers, maybe followed by normalization."""
 
-    def __init__(self, config: ACTConfig):
+    def __init__(self, config: ACTConfig, is_vae_encoder: bool = False):
```

Comment:
This just needs
```suggestion
    def __init__(self, config: ACTConfig, is_vae_encoder: bool=False):
```
to fix the linting error

Reply:
Pushed a fix with this formatting change